### PR TITLE
fix for rot lat-lon + 1/n tests

### PR DESCRIPTION
**Pull request by webisu**
added test from wmo rot lat-lon to lat-lon grid

### CODE REVIEW

**Comment by AlysonStahl-NOAA** [comment]
Is this the same as #221 ?

**Comment by webisu** [comment]
ip_fix_wne_1 is the fix and 1 test
ip_fix_wne_2 includes ip_fix_wne_1 and additional test

**Comment by AlysonStahl-NOAA** [comment]
They appear to be the exact same changes. Also, are you still doing work on these tests? The data files you added aren't used at all and you added an "exit 1" statement which is causing the tests to fail automatically.

File: tests/run_ipolates_tests.sh [hunk 1]
```diff
@@ -6,10 +6,10 @@
 set -e
 
 echo "create a new grib file with integer values"
-../wgrib2/wgrib2 data/gdas.t12z.pgrb2.1p00.anl.75r.grib2 -match "(UGRD|VGRD)" -rpn floor -set_scaling 0 0 -grib_out test.grb
+../wgrib2/wgrib2 data/gdas.t12z.pgrb2.1p00.anl.75r.grib2 -match "(UGRD|VGRD)" -rpn floor -set_scaling 0 0 -grib_out new_grid_test.grb
 
 echo "*** Testing conversion from earth to grid"
-../wgrib2/wgrib2 test.grb -new_grid_winds grid  \
+../wgrib2/wgrib2 new_grid_test.grb -new_grid_winds grid  \
```
Comment (edwardhartnett) [inline tests/run_ipolates_tests.sh:12]: Use the name of the test script in the name of any test files produced, like this:

run_ipolates_tests_new_grid_test.grb

Put the test script name first on all of them. Then, when looking at an ls of the test directory, all the test data produced by one test will be easy to see.

**Review by edwardhartnett — APPROVED** [review]

**Pull request merged**